docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -367,6 +367,49 @@ nations by population::
         )
       );
 
+.. _sqlserver-procedure-function:
+
+``procedure(varchar) -> table``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``procedure`` function allows you to run stored procedures on the underlying
+database directly. It requires syntax native to SQL Server, because the full query
+is pushed down and processed in SQL Server. In order to use this table function we
+need to set ``sqlserver.stored-procedure-table-function.enabled`` to ``true``.
```

Comment:
"we need to set ... to `true`" -> "set the ... catalog configuration property to `true`"